src/BaseApi.ts

```diff
@@ -40,6 +46,8 @@ export class BaseApi {
         method,
         body: JSON.stringify(body),
         headers,
+        /** @ts-expect-error we get types from DOM here, but we should use node types */
```

Comment:
This is messy because node v18 types don't include fetch, so I didn't fix this for now, but I should fix this